packages/babel-preset-env/src/utils.js

```diff
@@ -152,11 +152,11 @@ export function isPolyfillSource(source: ?string): boolean {
 }
 
 const modulePathMap = {
-  "regenerator-runtime": "regenerator-runtime/runtime",
+  "regenerator-runtime": "regenerator-runtime/runtime.js",
```

Comment:
`regenerator-runtime` works in browsers, so it currently works with AMD I think.

Reply:
`regenerator-runtime/runtime.js` does not use `define` to declare its name and dependencies, if any. So I guess it is likely that no one consume it as an AMD module.